src/scales/scale.time.js

```diff
@@ -71,7 +71,7 @@ function arrayUnique(items) {
 
 	for (i = 0, ilen = items.length; i < ilen; ++i) {
 		item = items[i];
-		if (!hash[item]) {
+		if (item !== null && !hash[item]) {
```

Comment:
Then that's not anymore `arrayUnique` if it filters `null` values. Can we make that check elsewhere without iterating the whole array?